README.md

```diff
@@ -252,6 +252,7 @@ List of projects that provide terminal user interfaces
 - [calcure](https://github.com/anufrievroman/calcure) Modern TUI calendar and task manager with minimal and customizable UI.
 - [calcurse](https://calcurse.org/) calendar and scheduling application for the command line
 - [fml](https://github.com/wick3dr0se/fml) :file_folder: A stupid simple, fast file manager written in BASH v4.2+
+- [fjira](https://github.com/mk-5/fjira) TUI application for Attlasian Jira
```

Comment:
Typo: that should be "Atlassian"

Reply:
@huyz https://github.com/rothgar/awesome-tuis/pulls here is a [fix](https://github.com/rothgar/awesome-tuis/pull/147)